MongoProject/pom.xml

```diff
@@ -25,7 +25,25 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		
+		<dependency>
+    <groupId>io.prometheus</groupId>
+    <artifactId>simpleclient_spring_boot</artifactId>
+    <version>0.0.21</version>
+	</dependency>
+	
+	<dependency>
+    <groupId>io.springfox</groupId>
+    <artifactId>springfox-swagger2</artifactId>
+    <version>3.0.0</version>
+	</dependency>
 
+<!-- https://mvnrepository.com/artifact/org.apache.tomcat/tomcat-jasper -->
+<dependency>
+    <groupId>org.apache.tomcat</groupId>
+    <artifactId>tomcat-jasper</artifactId>
```

Comment:
why tomcat-jasper is needed?

Reply:
I included it firstly as jasper is not included in spring web but now i checked tomcat core  is embedded in spring boot starter so i guess we don't need it.